CMakeLists.txt

```diff
@@ -52,6 +52,10 @@ file(MAKE_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
 
 set(CXX_TEST_PATH ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
 
+set(KWStyle_MAJOR_VERSION "1")
```

Comment:
To avoid duplication with the `project` command, can we use:

```
set(KWStyle_MAJOR_VERSION "${${PROJECT_NAME}_VERSION_MAJOR}")
[...]
```

?

Reply:
Thanks for this suggestion, it allowed me to simplify things in the updated commit.